src/components/Figure/Image.js

```diff
@@ -8,15 +8,13 @@ const styles = {
   })
 }
 
-const Image = ({ data: { src, alt }, attributes = {} }) => {
+const Image = ({ src, alt, attributes = {} }) => {
```

Comment:
Please also update 
TeaserFront/Image.js
TeaserFront/Split.js
back to the original src/alt props.